code/modules/clothing/chameleon.dm

```diff
@@ -69,6 +69,7 @@
 /datum/action/item_action/chameleon/change
 	name = "Chameleon Change"
 	var/list/chameleon_blacklist = list()
+	var/list/chameleon_blacklist_types = list()
```

Comment:
the var names are really confusing, plus if they are checking types make it a typecache!